decidim-elections/app/cells/decidim/elections/election_m_cell.rb

```diff
@@ -91,6 +91,23 @@ def footer_button_text
       def statuses
         []
       end
+
+      def needs_to_show_remaining_time?
+        model.end_time > Time.now.utc && model.end_time < 12.hours.from_now.utc
+      end
+
+      def remaining_time
+        minutes_to_end = ((model.end_time - Time.now.utc) / 60).floor
```

Comment:
Same as in previous comment 😁

```suggestion
        minutes_to_end = ((model.end_time - Time.current) / 60).floor
```